test.html

```diff
@@ -1,2 +1,2 @@
 <h1>Hello</h1>
-<h2>goodbye</h2>
\ No newline at end of file
+<h2>goodbye</h2>    
```

Comment:
James, why have you done a tab here? No need, please change.

Reply:
thats been done